doc/building.html

```diff
@@ -393,8 +393,7 @@ <h2 id="operating-system-requirements">Operating System
 to list successes or failures of building on different platforms.</p>
 <h3 id="windows">Windows</h3>
 <p>Windows XP is not a supported platform, but all newer Windows should
-be able to build the JDK. (Note: The Windows 32-bit x86 port is
-deprecated and may be removed in a future release.)</p>
+be able to build the JDK.</p>
```

Comment:
I'd delete the entire paragraph as I doubt versions immediately after XP can actually build the JDK.

Reply:
I say we do Windows docs cleanups separately from this PR: [JDK-8345166](https://bugs.openjdk.org/browse/JDK-8345166).